src/commands/PublishCommand.js

```diff
@@ -374,9 +375,8 @@ export default class PublishCommand extends Command {
     }
   }
 
-  updatePackageDepsObject(pkg, depsKey) {
+  updatePackageDepsObject(pkg, depsKey, exact) {
     const deps = pkg[depsKey];
-    const {exact} = this.getOptions();
```

Comment:
It was bugging me that we were calling this every time, even if there were no deps.